regex_checker.py

```
@@ -0,0 +1,114 @@
+    Fluentd形式の正規表現をPython形式に変換します。
+
+    Args:
+        pattern (str): Fluentd形式の正規表現パターン
+
+    Returns:
+        str: Python形式の正規表現パターン
+    """
+    # (?<name>pattern) を (?P<name>pattern) に変換
+    return re.sub(r'\(\?<([^>]+)>', r'(?P<\1>', pattern)
+
+def process_log_file(log_file: str, pattern: str, matched_output: str, unmatched_output: str, captured_output: str) -> Tuple[int, int, int, List[Dict]]:
```

Ruff (0.8.2)

Use `tuple` instead of `Tuple` for type annotation

Replace with `tuple`

(UP006)